src/Transports.AspNetCore/GraphQLHttpMiddleware.cs

```diff
@@ -89,30 +97,38 @@ private bool IsGraphQLRequest(HttpContext context)
                 }
             }
 
-            object userContext = null;
+            var opts = CreateDefaultExecutionOptions();
+            opts.Schema = schema;
+            opts.Query = gqlRequest.Query;
+            opts.OperationName = gqlRequest.OperationName;
+            opts.Inputs = gqlRequest.Variables.ToInputs();
+
             var userContextBuilder = context.RequestServices.GetService<IUserContextBuilder>();
-            if (userContextBuilder != null)
+            if (userContextBuilder != null) {
+                opts.UserContext = await userContextBuilder.BuildUserContext(context);
+            }
+
+            if (opts.ValidationRules == null)
             {
-                userContext = await userContextBuilder.BuildUserContext(context);
+                opts.ValidationRules = context.RequestServices.GetService<IEnumerable<IValidationRule>>();
```

Comment:
This is a change in behavior in that people will now be required to add all of the Core Rules to the DI Container.  Was that intentional?

Reply:
Fixed in 93b56cf